pandas/tests/extension/base/setitem.py

```diff
@@ -195,3 +196,14 @@ def test_setitem_preserves_views(self, data):
         data[0] = data[1]
         assert view1[0] == data[1]
         assert view2[0] == data[1]
+
+    def test_setitem_nullable_mask(self, data):
+        # GH 31446
+        # TODO: there is some issue with PandasArray, therefore,
```

Comment:
can you create an issue for this? (e.g. PandasArray support for setitem)

Reply:
will do

Reply:
@charlesdong1991 is there an opened issue for this yet? I haven't found it and it causes CI of geopandas to fail as contrary to the comment this test is not skipped.

Reply:
@martinfleis The issue for geopandas is not due to PandasArray not working, but rather because we need to update our `__setitem__` implementation (similar to how I updated the `__getitem__`), will do a PR for that.